unstructured/ingest/connector/delta_table.py

```diff
@@ -0,0 +1,124 @@
+import logging
+import os
+import uuid
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import pandas
+import pyarrow
+from deltalake import DeltaTable
+from pyarrow import RecordBatch
+
+from unstructured.ingest.interfaces import (
+    BaseConnector,
+    BaseConnectorConfig,
+    BaseIngestDoc,
+    ConnectorCleanupMixin,
+    IngestDocCleanupMixin,
+    StandardConnectorConfig,
+)
+from unstructured.ingest.logger import make_default_logger
+from unstructured.utils import requires_dependencies
+
+
+@dataclass
+class SimpleDeltaTableConfig(BaseConnectorConfig):
+    verbose: bool
+    table_uri: Union[str, Path]
+    version: Optional[int] = None
+    storage_options: Optional[Dict[str, str]] = None
+    without_files: bool = False
+    columns: Optional[List[str]] = None
+    batch_size: Optional[int] = None
+    logger: Optional[logging.Logger] = None
+
+    def get_logger(self) -> logging.Logger:
+        if self.logger:
+            return self.logger
+        return make_default_logger(logging.DEBUG if self.verbose else logging.INFO)
```

Comment:
maybe we should switch to just use the same default logger everywhere in ingest (rather than having this configurable in some)?

Reply:
I originally wanted to add this to the `BaseConnectorConfig` class, but since we're supporting python3.8,  this doesn't have support for the `kw_only` param in `@dataclass(kw_only=True)`, so doing something like this: 
```python
@dataclass
class BaseConnectorConfig(ABC):
    """Abstract definition on which to define connector-specific attributes."""

    verbose: bool = False
    logger: Optional[logging.Logger] = None

    def get_logger(self) -> logging.Logger:
        if self.logger:
            return self.logger
        self.logger = make_default_logger(logging.DEBUG if self.verbose else logging.INFO)
        return self.logger
```
results in: `error: Attributes without a default cannot follow attributes with one`

Reply:
oh I worded poorly, I meant why do we need this configurable at all? can we just use the same logger everywhere?

Reply:
When they used a shared logger, it ran into issues. I think because of the multiprocessing fork. It was dropping logs. This way each fork has a dedicated logger.

Reply:
are all of the other connectors that don't have this logic currently dropping logs?

Reply:
I haven't thoroughly checked. They might be. I couldn't narrow exactly when logs were dropped but it seemed having this solution prevented it.